operator/src/main/java/org/keycloak/operator/controllers/KeycloakDeploymentDependentResource.java

```diff
@@ -534,7 +532,7 @@ private List<EnvVar> getDefaultAndAdditionalEnvVars(Keycloak keycloakCR) {
             }
         }
 
-        envVars.add(new EnvVarBuilder().withName(POD_IP).withNewValueFrom().withNewFieldRef()
```

Comment:
There is a small chance this could be seen as a breaking change - if users assume that the POD_ID environment variable is always available.